recipes/default.rb

```diff
@@ -16,6 +16,12 @@
 # limitations under the License.
 #
 
+directory  "/var/chef/handlers" do
+  action :create
+  user "root"
+  group "root"
+end
```

Comment:
This resource needs to be executed in the Chef compile phase, as this is when the attempt to create the Chef Slack handler occurs, so this last line needs to look like this:

`end.run_action(:create)`